src/main/resources/db/changelog/changes/11-create-books_categories-table.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books_categories-table
+      author: ivan-diatliuk
+      changes:
+        - createTable:
+            tableName: books_categories
+            columns:
```

Comment:
add constraints to this table:
- PK (book_id, category_id)
- FK (book_id)
- FK (category_id)